src/spectro_inlets_quantification/sensitivity.py

```diff
@@ -218,13 +230,41 @@ def __len__(self) -> int:
         """Return the number of the SensitivityFactors in the object."""
         return len(self.sf_list)
 
-    def __add__(self, other: "SensitivityList") -> "SensitivityList":
-        """Return a `SensitivityList` with the sensitivity factors of itself and ``other``."""
+    def __add__(self, other: [SensitivityFactor or "SensitivityList"]) -> "SensitivityList":
```

Comment:
I was not aware that you could write the types like that. Are you sure it does what you want it to? Since SensitivityFactor is a normal class I would expect this to return just [SensitivityFactor], which I don't think does what you want. The canonical way I know of would be `Union[SensitivityFactor, "SensitivityList"]`

Reply:
No, I was not sure. This looks like mistake on my part. Switched to the cononical way.